packages/node-cli/src/services/AccountCredentialsService.js

```diff
@@ -0,0 +1,62 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+'use strict';
+const os = require('os');
+const NodeTranslationService = require('./NodeTranslationService');
+
+const {
+	ACCOUNT_TYPE,
+	COMMAND_MANAGE_ACCOUNT: {
+		MESSAGES: { ACCOUNT_INFO },
+		QUESTIONS_CHOICES,
+	},
+} = require('./TranslationKeys');
+
+const SANDBOX_ACCOUNT_ID_REGEX_PATTERN = '.+_SB\\d*$';
+const RELEASE_PREVIEW_ACCOUNT_ID_REGEX_PATTERN = '.+_RP\\d*$';
+
+module.exports = class AccountCredentialsService {
```

Comment:
I see this class more like a transformer/formater than a Service. I would prefer to name it AccountCredentialsFormater. And then name the methods something similar to, getAccountInfoString, and getAuthIdInfoString.
I also think it will be nice to move some of the TranslationKeys out of the COMMAND_MANAGE_ACCOUNT, because I think they are only used here.